components/authz-service/server/v2/rules_property_test.go

```diff
@@ -33,25 +37,25 @@ func TestCreateRuleProperties(t *testing.T) {
 			api.ProjectRuleConditionAttributes_POLICY_GROUP,
 			api.ProjectRuleConditionAttributes_POLICY_NAME,
 		),
-		"Values": gen.SliceOf(gen.AnyString()),
+		"Values": gen.SliceOf(gen.UnicodeString(graphicRange)),
```

Comment:
Should we be generating an `Operator` as well?

Reply:
Yep, I'm gonna do a follow up PR where I add a bunch of test coverage. Just getting this in with the DB changes and will pick up after we wrap up everything we need to see projects + ingest work e2e.